api/src/main/java/io/opentelemetry/metrics/DoubleSumObserver.java

```diff
@@ -16,33 +16,42 @@
 
 package io.opentelemetry.metrics;
 
-import io.opentelemetry.metrics.DoubleSumObserver.ResultDoubleObserver;
+import io.opentelemetry.metrics.DoubleSumObserver.ResultDoubleSumObserver;
 import java.util.Map;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Observer to report pre-aggregated metrics with double value.
+ * {@code SumObserver} is the asynchronous instrument corresponding to Counter, used to capture a
+ * monotonic sum with Observe(sum).
+ *
+ * <p>"Sum" appears in the name to remind that it is used to capture sums directly. Use a
+ * SumObserver to capture any value that starts at zero and rises throughout the process lifetime
+ * and never falls.
+ *
+ * <p>A {@code SumObserver} is a good choice in situations where a measurement is expensive to
+ * compute, such that it would be wasteful to compute on every request.
  *
  * <p>Example:
  *
  * <pre>{@code
  * class YourClass {
  *
  *   private static final Meter meter = OpenTelemetry.getMeterRegistry().get("my_library_name");
- *   private static final DoubleSumObserver observer =
+ *   private static final DoubleSumObserver cpuObserver =
  *       meter.
- *           .doubleSumObserverBuilder("doWork_latency")
- *           .setDescription("gRPC Latency")
+ *           .doubleSumObserverBuilder("cpu_usage")
```

Comment:
CPU usage as provided by proc or other sources is a classic monotonic always increasing sum, it is the total time in jiffrey (probably I misspelled this) since the beginning of the process